ext/spl/tests/SplFixedArray_serialize.phpt

```diff
@@ -24,10 +26,23 @@ var_dump($unser[0], $unser[1], $unser[2], $unser[3], $unser[4]);
 
 $unser[4] = 'quux';
 var_dump($unser[4]);
+var_dump($unser->foo);
+
+$array = new SplFixedArray(5);
+$array->__unserialize([
+    [null],
```

Comment:
1. Change this to anything other than null to make the test indicate if this actually starts overwriting anything
2. Add a comment like `// __unserialize` is a no-op on a non-empty SplFixedArray
3. nit: Can use a smaller array size in constructor for shorter output, above lines differ in that they use array[4]